docs/abi.md

```diff
@@ -214,8 +211,18 @@ Fills a buffer with random bytes.
 
 The host functions described in the previous section allow opaque blobs of data
 to be exchanged (along with handles to other channels). When communicating with
-the pseudo-Nodes provided by the Oak system, these opaque blobs of data are
-defined to take the form of serialized protocol buffer messages. These include:
+the Runtime, these opaque blobs of data are defined to take the form of
+serialized protocol buffer messages:
+
+- The label values included on Node and channel creation operations are in the
```

Comment:
FWIW, this section seems to repeat what is already defined in the various descriptions above, but fine to keep around as extra clarification.

Reply:
That's deliberate, this section is trying to describe conventions that apply across the whole ABI surface.

However, now that it doesn't just apply to pseudo-Node communication but to core host functions too, I think this section belongs up next to the "Integer Types" section.